scala/scala_maven_import_external.bzl

```diff
@@ -64,7 +64,7 @@ def _jvm_import_external(repository_ctx):
     if (repository_ctx.attr.generated_linkable_rule_name and
         not repository_ctx.attr.neverlink):
         fail("Only use generated_linkable_rule_name if neverlink is set")
-    name = repository_ctx.attr.generated_rule_name or repository_ctx.name
+    name = repository_ctx.attr.repo_name or repository_ctx.attr.generated_rule_name or repository_ctx.name
```

Comment:
names of repositories issued by bzlmod module extensions will look like:
`_main~non_module_deps~new_repo`
this is why I added explicit parameter to achieve: `@new_repo`